distributed-job-manager/src/test/java/ru/fix/distributed/job/manager/WorkPooledMultiJobSharingIT.java

```diff
@@ -15,74 +12,56 @@
 import java.util.HashSet;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
 
 import static org.mockito.Mockito.*;
 
 class WorkPooledMultiJobSharingIT extends AbstractJobManagerTest {
 
-    private final WorkItemMonitor monitor = mock(WorkItemMonitor.class);
```

Comment:
I removed WorkItemMonitor class from AbstractJobManagerTest because it used only in this test and can be easily replaced with built-in Consumer class